base/applications/taskmgr/graphctl.c

```diff
@@ -12,666 +12,363 @@
 
 WNDPROC OldGraphCtrlWndProc;
 
-static void GraphCtrl_Init(TGraphCtrl* this)
+void TmgcCreate(PTM_GRAPH_CONTROL inst, HWND hWnd, HWND hParentWnd, PTM_FORMAT fmt)
```

Comment:
Regarding your failure question, here is a suggestion: create function returns TRUE/FALSE depending on whether the creation succeeded or failed:
```suggestion
BOOL TmgcCreate(PTM_GRAPH_CONTROL inst, HWND hWnd, HWND hParentWnd, PTM_FORMAT fmt)
```